src/routes/concepts/refs.mdx

```diff
@@ -7,7 +7,9 @@ They are particularly useful when you need to access the DOM nodes directly or i
 
 ## Accessing DOM elements
 
-One way of accessing DOM elements is through [element selectors](https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors) such as [`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) or [`document.getElementById`](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById).
+One way of accessing DOM elements is through [element selectors](https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors) 
+such as [`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) or 
+[`document.getElementById`](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById).
```

Comment:
```suggestion
One way of accessing DOM elements is through [element selectors](https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors) such as [`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) or [`document.getElementById`](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById).
```

Reply:
@LadyBluenotes is there a reason you want to keep it as one long line? I'm pretty sure it renders the same and makes the markup file easier to read since you don't have to scroll horizontally as much

Reply:
It's in our writing guide that each sentence get it's own line. Hence, keeping the conventions consistent across the board.

https://github.com/solidjs/solid-docs-next/blob/main/WRITING.md#organization